posts/inside-rust/2019-12-09-announcing-the-docsrs-team.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: post
+title: "Announcing the Docs.rs Team"
+author: QuietMisdreavus
+team: The Rustdoc Team <https://www.rust-lang.org/governance/teams/dev-tools#rustdoc>
+---
+
+Today we're announcing a brand new team: The Docs.rs Team!
+
+Previously, [Docs.rs] has been managed by the Rustdoc Team, as many of the initial concerns of
+Docs.rs were shared by Rustdoc as Docs.rs was being brought into team maintainership. However, as
+time went on, those concerns started to diverge more and more, and so did the people who contributed
+to either tool.
+
+The new Docs.rs Team will be responsible for the operations and development for [Docs.rs], leaving
+the Rustdoc Team to be responsible for the Rustdoc tool itself. [@QuietMisdreavus] will be leading
+the new Docs.rs Team, leaving [@GuillaumeGomez] to lead the Rustdoc Team.
+
+Joining QuietMisdreavus on the Docs.rs Team is GuillaumeGomez, coordinating work between Rustdoc and
```

Comment:
You didn't put my username as a link here. :)

Reply:
We were both linked in the previous paragraph. 😉

Reply:
You forgot to link yourself too! :o

Reply:
(I like to link everything hehe)

Reply:
I don't like to have a lot of redundant links, especially when you can look one paragraph up to find the same term and see a link there. Personal preference, i guess.